client/src/Client.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+
+function checkStatus(response) {
```

Comment:
better to use [Axios](https://github.com/axios/axios) to handle asynchronous calls to the server.

Reply:
Done